test/test_db.py

```diff
@@ -240,6 +240,30 @@ def test_get_channel_value(db):
     assert result == 'zxcv'
 
 
+def test_set_plugin_value(db):
+    conn = sqlite3.connect(db_filename)
+    db.set_plugin_value('#asdf', 'qwer', 'zxcv')
```

Comment:
Plugin named `#asdf`? (repeated below)

Reply:
I can make that say whatever, however,,, the dB tests still needs to be updated for sqlalchemy instead of directly accessing sqlite.